scriptworker/cot/verify.py

```diff
@@ -1110,21 +1110,26 @@ def _get_additional_github_releases_jsone_context(parent_link, decision_link):
         jsone_context.update(
             _get_additional_github_releases_jsone_context(parent_link, decision_link)
         )
+
```

Comment:
I had to look this up, since I thought cot_product would say 'mobile' for Firefox-for-Android (from hg), I was wrong.

Reply:
I'm sorry for the confusion. We currently have 3 different `cot_product`s:

* `firefox` -> for all mozilla-* hg repos
* `comm` -> for comm-* hg repos
* `mobile` -> for all new mobile projects that are on GitHub.

I'll try to make this clearer in the tests.